DTM/Ponylands2_DTM/map.json

```diff
@@ -0,0 +1,130 @@
+{
+	"name": "Ponylands 2",
+	"authors": ["Clijmart, Xuph, KirillSemyonkin"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "Blue",
+			"name": "Blue",
+			"color": "blue",
+			"max": 50,
+			"min": 1
+		},
+		{
+			"id": "Red",
+			"name": "Red",
+			"color": "red",
+			"max": 50,
+			"min": 1
+		}
+	],
+	"spawns": [
+		{"teams": ["spectators"], "coords": "0, 33, 0"},
+		{"teams": ["Blue"], "coords": "-97.5,42,1.5,-90,0"},
```

Comment:
Same with these 2 teams, you aren't putting a space after the commas.

Reply:
Fixed them!